src/microbe_stage/editor/MicrobeEditor.tscn

```diff
@@ -1562,8 +1562,13 @@ custom_colors/font_color_pressed = Color( 0, 0, 0, 1 )
 toggle_mode = true
 action_mode = 0
 enabled_focus_mode = 0
+<<<<<<< HEAD
+group = SubResource( 21 )
+text = "MEMBRANE"
+=======
 group = SubResource( 15 )
 text = "APPEARANCE"
+>>>>>>> master
```

Comment:
```suggestion
group = SubResource( 15 )
text = "MEMBRANE"
```

Reply:
Strange, my editor wouldn't point that one out to me at all, so I had to find it manually.

Reply:
It isn't fixed, there is still one bad line (`>>>>>>> master`). Fix the file via a text editor.

Reply:
Should be gone now, apologies for the time.